extra_exercises/test_reverse_linked_list.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from linked_list.linked_list import LinkedList
+from reverse_linked_list import reverse_with_list
+
+class ReverseLinkedListTest(unittest.TestCase):
+
+    def test_reverse_linked_list_with_list(self):
+        linked_list = LinkedList()
+        linked_list.append(9)
+        linked_list.append(5)
+        linked_list.append(15)
+        linked_list.append(2)
+        self.assertIsNotNone(linked_list)
```

Comment:
you're not testing the linked list creation here, so this test shouldn't be here